File: pset1/mario/mario.c
# include<stdio.h>
# include<cs50.h>

void print_pyramid(int height);

int main(void){

    int height;

    do{
        height = get_int("Height: ");

    }while(height<1 || height>8);

    // printf("height: %i\n",height);
    print_pyramid(height);
}

void print_pyramid(int height){
    int n =1;
    // while(n<=height){

    //     for(int i=0;i<height;i++){

    //         for(int j=0;j<n;j++){

    //             printf("#");
    //         }
    //         printf("\n");
    //         n+=1;

    //     }
    // }

    while(n<=height){

        for(int i=0;i<height;i++){

            for(int j=0;j<height;j++){

                if(j<(height-i-1)){
                    printf(" ");
                }else{
                    printf("#");
                }
            }
            printf("\n");
            n+=1;
        }
    }

}

File: pset6/mario/more/mario.py
from cs50 import get_int

while(True):
    number = get_int("Height: ")
    if(number >=1 and number <=8):
        break


for i in range(number):
    # Printing right pyramid
    for j in range(number):
        if(j < number - i - 1):
            print(" ", end ="")
        else:
            print("#", end="")

    print(" ", end="")

    # Printing left pyramid
    for k in range(i+1): # k runs from 0 to i-1
        print("#", end="")

    print()